core/templates/dev/head/css/oppia.css

```diff
@@ -3884,7 +3885,7 @@ md-card.preview-conversation-skin-supplemental-card {
 }
 
 .oppia-dashboard-card-view-item {
-  height: 250px;
+  height: inherit.oppia-dashboard-statistics-card-link;
```

Comment:
I did not know you could do this in CSS. Can you point me to some documentation about this? (I searched online but could not find it.)